pages/pipelines/waterfall.md

```diff
@@ -0,0 +1,34 @@
+# Waterfall view
+
+> 📘 Business/Enterprise feature
+> Waterfall is only available on [Business or Enterprise](https://buildkite.com/pricing) plans.
+
+## Overview
+
+Waterfall view allows you to see build data as a waterfall chart, providing enhanced visibility into your build's job processes, durations and dependencies.
+
+To access waterfall view:
+
+1. Navigate to any build page.
+1. Select _View_.
+1. Select _Waterfall_ from the dropdown menu.
+
+Waterfall view displays data for finished steps only.
+
+<!-- TODO: SCREENSHOT OF BASIC ROW WITH POPOVER GOES HERE -->
+
+Most rows will show bars with three coloured sections.
```

Comment:
Just breaking up the paragraph into a list so it's easier to scan for information.
Not sure if we can be more explicit about what this "time" is? 
> and the time the agent started running the job.
I have included some clumsily worded phrase about it being a timestamp - is this factual? If you want to change it feel free.

Also, I think we use US English for Grey? 🤔

```suggestion
Most rows will show bars with three colored sections:
1. Gray: time the job spent waiting for an agent to be assigned.
1. Yellow: time elapsed since the agent was assigned, and a timestamp when the agent started running the job.
1. Green or Red: total job time, from when the agent started running the job to when it finished running the job. Displayed as green for _passed_ job or red for _failed_ job.
```

Reply:
ahhh yep, we do. I always forget to do US spelling 😅

Reply:
>I have included some clumsily worded phrase about it being a timestamp - is this factual? If you want to change it feel free.

So we don't show a timestamp per se - it's a bar split into three colored sections, with each section representing a duration. The time displayed next to the bar is a sum of the three durations, giving the total (i.e. including waiting) time the job or step took to conclude. 

Sorry, I should have included a screenshot for reference to make this clearer. Here's an example: 
![Screenshot 2023-09-26 at 11 13 14 AM](https://github.com/buildkite/docs/assets/10970711/59be0a31-b361-467f-a79f-6bf25e9ef7fb)

Here's what it looks like in the popover on hover (note that because of rounding, the numbers are a bit different): 
![Screenshot 2023-09-26 at 11 13 59 AM](https://github.com/buildkite/docs/assets/10970711/4b298fbe-85a9-4dc0-a069-480a654fc20b)

What do you think about the following: 

```
1. Yellow: time elapsed since the agent was assigned, up until the time the agent started running the job
1. Green or Red: time the agent spent running the job. Displayed as green for _passed_ job or red for _failed_ job.
```

Reply:
Yep! That looks good! ✅✨